packages/schemas/tables/connectors.sql

```diff
@@ -0,0 +1,12 @@
+create type connector_type as enum ('SMS', 'Email', 'Social');
+
+create table connectors (
+  id varchar(128) not null,
+  identifier varchar(256) not null unique,
+  enable boolean not null default TRUE,
```

Comment:
discussion: should we add prefix `is`, `has` for boolean fields? @logto-io/eng

Reply:
I prefer boolean fields with prefixes.

Reply:
any ideas? @simeng-li @IceHe

Reply:
prefer `enabled`